toolbox/workflow/configs/debian-bullseye.config

```diff
@@ -137,6 +137,12 @@ devices:
   driver_attachment: "kernel"
   spdk_conf:
 
+- uri: "2GB"
+  nsid: 1
+  labels: ["dev", "bdev", "nvm", "ramdisk"]
+  driver_attachment: "kernel"
+  spdk_conf:
```

Comment:
Is it possible to just omit these requirement?

Reply:
The driver-helper needs to know about the "driver_attachment". So that one needs to be there.
The "spdk_conf" could be omitted.